python/source/org/libreoffice/ide/eclipse/python/messages.properties

```diff
@@ -0,0 +1,6 @@
+Language.JavaNatureSet=Java nature set
```

Comment:
Should be python nature

Reply:
This has been changed to 'Language.PythonNatureSet=Python nature set'